### PR TITLE
Update custom-functions-json.md

### PR DESCRIPTION
**Pull request by keyur32**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812010136177252-593%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kSuEkKTwxV2BpCgTOuMr0Bh%2Bi6Ah9XQGpG3h1tCfYzc%3D&st=2018-12-01T01%3A37%3A51Z&se=2019-01-01T01%3A42%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-json.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ The `options` object enables you to customize some aspects of how and when Excel
...
-|  `volatile`  | boolean | No <br/><br/>Default value is `false`. | If `true`, the function will recalculate each time Excel recalculates, instead of only when the formula's dependent values have changed. A function cannot be both streaming and volatile. If the `stream` and `volatile` properties are both set to `true`, the volatile option will be ignored. |
+|  `volatile`  | boolean | No <br/><br/>Default value is `false`. | **Coming to January Office Insiders Build** <br /><br /> If `true`, the function will recalculate each time Excel recalculates, instead of only when the formula's dependent values have changed. A function cannot be both streaming and volatile. If the `stream` and `volatile` properties are both set to `true`, the volatile option will be ignored. |
```
Comment (mscharlock) [inline docs/excel/custom-functions-json.md]: If this is now in Jan - does this mean that the docs should take it out until Jan or do you want people to know in advance for some reason? Does including it currently break things? My understanding the last time I tried it was that it is ok except that the intelliSense doesn't work for it yet.

Comment (mscharlock) [reply docs/excel/custom-functions-json.md]: Checked with Reeza, I will update to take out and then merge this PR

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C5%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812050017584504-593%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wmjtyIm4P0GLHf8Ljg3YC5nPX7S92bSLQ3oKexDC99k%3D&st=2018-12-05T00%3A14%3A09Z&se=2019-01-05T00%3A19%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**